client/notifications/style.scss

```diff
@@ -134,7 +134,7 @@
 			background: transparent;
 		}
 		ul,
-		ol {
+		ol:not( .wpnc__ol--outside ) {
```

Comment:
It's too bad the `:not()` is needed here and the `wpnc__ol--outside` class can't just gracefully override this style. But I'm guessing the use of the `#wpnc-panel` id selector bumps the specificity up too high. I guess you could add `!important` to the `ol.wpnc__ol--outside` selector 🤮

Reply:
Yeah the priority of `#wpnc-panel` id selector is too hight 😞